packages/tables/resources/lang/ka/table.php

```diff
@@ -2,8 +2,28 @@
 
 return [
 
+    'columns' => [
+
+        'tags' => [
+            'more' => 'და კიდევ :count',
+        ],
+
+        'messages' => [
+            'copied' => 'დაკოპირებულია',
+        ],
+
+    ],
+
     'fields' => [
 
+        'bulk_select_page' => [
+            'label' => 'Select/deselect all items for bulk actions.',
+        ],
+
+        'bulk_select_record' => [
+            'label' => 'Select/deselect item :key for bulk actions.',
```

Comment:
Could you still translate this value?

Reply:
I'll merge this PR, but feel free to translate those missing strings in a new PR, @Stichoza.